packages/server/graphql/mutations/addTeam.ts

```diff
@@ -41,15 +40,8 @@ export default {
       }
 
       // VALIDATION
-      const orgTeams = await r
-        .table('Team')
-        .getAll(orgId, {index: 'orgId'})
-        .run()
-
-      const orgTeamNames = [] as string[]
-      orgTeams.forEach((team) => {
-        if (!team.isArchived) orgTeamNames.push(team.name)
-      })
+      const orgTeams = await getTeamsByOrgIds(orgId, {isArchived: false})
```

Comment:
So...I can't create a team on this PR :astonished: 

calling `addTeam` throws an error because getTeamsByOrgIds takes an array as the first param. 
While the fix is easy, I'm a little concerned that this broken code went through 2 reviews approvals.
In the future can we be sure to test touched files? Some things are sure to slip through the cracks, and that's OK! But if 3 people test all the changed files we should be able to avoid broken stuff landing in prod.

@tianrunhe @tiffanyhan

Reply:
This is my bad. Apologize! Here's the retro:  
  
I checked the references for `getTeamsByOrgId` and saw two patterns  
1. places where `getTeamsByOrgIds` gets called directly
2. it's used in `teamsByOrgId` as a foreign data loader and the data loader is then used in a lot of places.

I tested everything in 2. and that's how I build the test cases. I did not go through all use cases in 1. and that's how `addTeam` get slipped.

In the future I think we do not only need to test all touched files, but also references to changed methods.